### DISCUSSION TITLE
Manual Mesh on Ender 3 v2 slows down retraction signifficantly

### DISCUSSION THREAD

**Discussion by BR4SH** [Jyers/Marlin#655] [Q&A] [answered]
Now that my machine is running Manual Mesh, the retraction speeds are much slower, which causes the nozzle to idle for almost 2 seconds before and after a single extrusion. This results in warping and melting from heat, as well as oozing.

Is there a setting to change the retraction speed value in the firmware? Cura settings haven't changed at all.

**Comment by Jyers** [comment, marked as answer]
Are you still seeing this issue? You could try checking your max speed settings in the motion menu.

**Reply by BR4SH** [reply]
Currently at a default of 25. How high do you think I should crank it? Keeping in mind we're using a Bondtech BGM extruder.

**Reply by Jyers** [reply]
Check your slicer for your retraction speed, then you can set it to that.

**Reply by BR4SH** [reply]
I set it to 50 (max allowed) and it was retracting really fast, which you could tell by that whining noise the motor makes when it pulls the filament back. Dropped it to 40 and now it looks like we're back on track. No noise, quick retractions.

Thanks for your help.

**Reply by Jyers** [reply]
I'd recommend tuning the retractions in your slicer as apposed to capping them using the max setting. That way you have the real retraction speed set in the slicer and not some random high number that's getting reduced by the firmware anyway.

**Reply by BR4SH** [reply]
I'm not sure how to dial-in an accurate 1x1 retraction value. The slicer is using a retraction speed of 50, which did well pre-firmware update, so I just guestimated the Max Speed setting based on the printer's behavior and print result.

**Reply by Jyers** [reply]
You're old firmware probably had the max speed around 40 then. I'd just recommend setting your slicer retraction to 40 as well.

**Reply by BR4SH** [reply]
Will do. Your help is really appreciated.
